WebSecurityScanner/tests/Unit/V1beta/WebSecurityScannerClientTest.php

```diff
@@ -22,72 +22,63 @@
 
 namespace Google\Cloud\WebSecurityScanner\Tests\Unit\V1beta;
 
-use Google\Cloud\WebSecurityScanner\V1beta\WebSecurityScannerClient;
 use Google\ApiCore\ApiException;
+
```

Comment:
nit: not a huge fan of these extra newlines

Reply:
This is automatically added by the linter, and changing this could potentially remove other newlines or shift indents around (due to bugs in the third-party linter).